src/Slub/Domain/Entity/PR/PR.php

```diff
@@ -174,6 +182,12 @@ public static function fromNormalized(array $normalizedPR): self
         $putToReviewAt = PutToReviewAt::fromTimestamp($normalizedPR[self::PUT_TO_REVIEW_AT]);
         $closedAt = ClosedAt::fromTimestampIfAny($normalizedPR[self::CLOSED_AT]);
 
+        // Protect from BC break
+        $isLarge = false;
+        if (isset($normalizedPR[self::IS_LARGE_KEY])) {
```

Comment:
So we have a new field in the PR table ? we need to add this column:
- Via a migration in the "Migrations" folder
- in the repository, we need to update sql queries to save/fetch the data
- when installing the slub db, we need to add this column as well